src/System.Text.Json/src/System/Text/Json/JsonConstants.cs

```diff
@@ -66,6 +66,7 @@ internal static class JsonConstants
         public const int MaximumFormatSingleLength = 128;  // default (i.e. 'G'), using 128 (rather than say 32) to be future-proof.
         public const int MaximumFormatDecimalLength = 31; // default (i.e. 'G')
         public const int MaximumFormatGuidLength = 36;    // default (i.e. 'D'), 8 + 4 + 4 + 4 + 12 + 4 for the hyphens (e.g. 094ffa0a-0442-494d-b452-04003fa755cc)
```

Comment:
nit: `MaximumFormattedGuidLength` (not `Format`)

Reply:
Keeping as is for consistency with other similar constants.